howtos/release-procedure.md

```diff
@@ -27,12 +27,18 @@ To prepare for a minor release, you'll need to create a new verison.
 
 ### Create new version
 
-Technically, the current contents are frozen in `docs` and copied to `versioned_docs` with the corresponding version. The process can be triggered by this Docusaurus command:
-
-```bash
-npm run docusaurus docs:version 8.1
-
-```
+Technically, the current contents are frozen in `docs` and copied to `versioned_docs` with the corresponding version.
+
+(TODO: expand these notes)
+
+- Link to multi-instance version explanation to explain why the process is more involved now
+- Step 1: make the ./hacks/cutNewVersions shell file executable, if it isn't already:
+  - `chmod +x ./hacks/cutNewVersions.sh`
+- Step 2: Update the versions in ./hacks/cutNewVersions.sh
+- Step 3: run cutNewVersions:
+  - `./hacks/cutNewVersions.sh`
+- Step 4: Manually update src/mdx/expandVersionedUrl.js to include new version mapping
```

Comment:
I'd _like_ to automate this step but if I don't get there, this is a pretty small manual thing to have to do. 🤷

Reply:
👍